packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -274,8 +255,6 @@ export async function copyProjectTemplateAndReplace(
     {
       id: options.useWinUI3 ? 'Microsoft.WinUI' : 'Microsoft.UI.Xaml',
       version: options.useWinUI3 ? winui3Version : winui2xVersion,
-      hasProps: false, // WinUI/MUX props and targets get handled by RNW's WinUI.props.
```

Comment:
What effect does removing this have?

Reply:
It was used to inject the `packages.config`-derived logic into the templates.

Using `PackageReference` renders those fragments unnecessary.

Reply:
These changes are perfect. It was to declare that some nuget packages had a props and targets that needed to be manually imported...